vm/src/vm.rs

```diff
@@ -250,6 +250,10 @@ impl VirtualMachine {
         syntax_error
     }
 
+    pub fn new_scope(&self) -> Scope {
```

Comment:
Please rename this function to reflect that it creates a scope and adds builtins. Something like: `new_scope_with_builtins`.

Reply:
Fixed it